pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -102,11 +101,11 @@ def __init__(
         self._distrib_type = None
 
         self.num_processes = num_processes
-        self.tpu_cores = device_parser.parse_tpu_cores(tpu_cores)
+        self.gpus = gpus
+        self.tpu_cores = tpu_cores
+        self.parallel_device_ids = gpu_ids
```

Comment:
missing explanation in docs what it does...

Reply:
They are for internal logic, but as we write new code we should still try to document/comment it for other contributors/ourselves in the future :)

Reply:
@Borda @justusschock That's a fair point! Have added a comment to help code walkers 🦮

Reply:
maybe include GPU in the name, so there is no confusion with IPU/TPU device

Reply:
@Borda That would be a breaking change. It's a refactor PR.
I did that in #7798, I will revisit that after #7808